app/assets/javascripts/host_edit_puppetclasses.js

```diff
@@ -0,0 +1,6 @@
+$(document).on('ContentLoad', function(){
```

Comment:
New JS files also need to be listed in config/environments/production.rb for precompilation unless only required by another.